src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      class="box__search1"
+      data-qa="big"
+    >
```

Comment:
Consider using more meaningful class names that represent the content rather than the styles or tag names. The class name 'box__search1' does not convey the purpose of the element. A better name might be 'search-form' or 'search-box', depending on the intended use.